src/guides/v2.4/graphql/queries/store-config.md

```diff
@@ -185,6 +175,36 @@ The following query returns information about the store's catalog configuration.
 }
 ```
 
+### Query a store's Customer configuration
```

Comment:
Just to keep the style the same for all titles 

```suggestion
### Query a store's customer configuration
```

Reply:
**fixed**